robot/util/emailer/package-lock.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "lidar",
```

Comment:
funny that this was auto generated, can we change it to something like `emailer`? or maybe `ip_emailer`?